TombEngine/Scripting/Internal/TEN/Input/InputHandler.cpp

```diff
@@ -28,6 +29,40 @@ namespace TEN::Scripting::Input
 		return true;
 	}
 
+	/// Get the analog value of an action key.
+	// @function GetAnalogKeyValue
+	// @tparam Input.ActionID actionID Action ID to query.
+	// @treturn float Analog value in the range [0, 1].
+	static float GetAnalogKeyValue(int actionID)
```

Comment:
Can you explain the use case for `GetAnalogKeyValue`? In which cases action key may return analog value?

Reply:
It will return the analog state of an action mapped to an analog input event (triggers, sticks, mouse).